src/CMakeLists.txt

```diff
@@ -497,10 +497,9 @@ FOREACH (name ${XSAC_SRC})
 
     ADD_CUSTOM_COMMAND (
         OUTPUT "${dir}/${dst}.sac"
-        # FIXME it would be better to call saccc like it is done currently
-        #       $(SACCC) $(SAC2C) prog seq -E "-x c" -I. $(DEFS)
         COMMAND
-            ${SAC2C} -cc ccprog -Xc -E -Xc "-x c" -noprelude -o /dev/stdout "${src}" > "${dir}/${dst}.sac"
+            # sac2c calls the preprocessor and the '-cppI's are there anyways!
+            cp "${src}" "${dir}/${dst}.sac" # sac2c calls the preprocessor and the 
```

Comment:
Looks like part of a previous comment remains in this line.

```
cp "${src}" "${dir}/${dst}.sac" # sac2c calls the preprocessor and the
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

I suspect it was intended to be removed, as it is (almost) the same as the comment in the line above.

Reply:
indeed :-(